src/components/Sponsors/index.js

```diff
@@ -1,94 +1,115 @@
 import React, { Component } from "react";
 
-import { Row, Col } from "react-bootstrap";
+import { Container, Row, Col, Image } from "react-bootstrap";
 
-import phi_logo from "../../assets/sponsor_logos/phi_logo.png";
-import ldss_logo from "../../assets/sponsor_logos/ldss_logo.png";
-import fossa_logo from "../../assets/sponsor_logos/fossa_logo.png";
-import df_logo from "../../assets/sponsor_logos/df_logo.png";
-import dspace_logo from "../../assets/sponsor_logos/dspace_logo.png";
-import next_canada_logo from "../../assets/sponsor_logos/next_canada_logo.png";
-import zebu_logo from "../../assets/sponsor_logos/zebu_logo.png";
-import camplete_logo from "../../assets/sponsor_logos/camplete_logo.svg";
+import caa_logo from "../../assets/sponsor_logos_2020/caa_logo.jpg";
+import digital_ocean_logo from "../../assets/sponsor_logos_2020/digital_ocean_logo.png";
+import jtdc_logo from "../../assets/sponsor_logos_2020/jtdc_logo.png";
+import shopify_logo from "../../assets/sponsor_logos_2020/shopify_logo.png";
+import sunlife_logo from "../../assets/sponsor_logos_2020/sunlife_logo.jpg";
+import uphabit_logo from "../../assets/sponsor_logos_2020/uphabit_logo.png";
+import voiceflow_logo from "../../assets/sponsor_logos_2020/voiceflow_logo.png";
 
 import styles from "./styles.module.css";
 
-export default class Sponsors extends Component {
+export default class PreviousSponsors extends Component {
```

Comment:
I think this is supposed to be "class Sponsors" and not "class PreviousSponsors"

Reply:
Good catch!